### ISSUE TITLE
ModuleNotFoundError: No module named 'wikipedia2vec.dictionary'

### ISSUE THREAD

**Issue by AMParanoid** [wikipedia2vec/wikipedia2vec#41]
Can't run 20 newsgroups test: 'python main.py train-classifier enwiki_20180420_lg1_300d.pkl enwiki_20180420_entity_linker.pkl --dataset=20ng'

**Comment by ikuyamada** [comment]
It seems that your Python env does not have Wikipedia2vec package. 
As explained in the README, it is required to install packages listed in requirements.txt before running main.py.

**Comment by AMParanoid** [comment]
Hi ikuyamada!

I installed, first thing I did.

running pip install wikipedia2vec returns:

Requirement already satisfied: wikipedia2vec in c:\users\amparanoid\documents\projetos\cits\wik\wikipedia2vec (1.0.4)

**Comment by ikuyamada** [comment]
Thanks for the prompt reply. Does the current directory contain a directory named  wikipedia2vec? Python may uses the code in the directory instead of the pip-installed package.

**Comment by AMParanoid** [comment]
Folder already exists:

'c:\Users\AMParanoid\Documents\Projetos\CITS\wik\wikipedia2vec\wikipedia2vec', containing:
__init__.py  cli.py          
dictionary.pyx  
dump_db.pyx     
link_graph.pyx  
mention_db.pyx  
wikipedia2vec.pyx
__pycache__  
dictionary.pxd  
dump_db.pxd     
link_graph.pxd  
mention_db.pxd  
utils

**Comment by ikuyamada** [comment]
Did you run main.py in *wikipedia2vec/examples/text_classification* directory? If so, does the directory contain a directory named wikipedia2vec?

**Comment by AMParanoid** [comment]
Yes I did:

c:\Users\AMParanoid\Documents\Projetos\CITS\wik\wikipedia2vec\examples\text_classification>python main.py train-classifier enwiki_20180420_lg1_300d.pkl enwiki_20180420_entity_linker.pkl --dataset=20ng
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    from wikipedia2vec.dictionary import Dictionary
  File "c:\Users\AMParanoid\Documents\Projetos\CITS\wik\wikipedia2vec\examples\text_classification\wikipedia2vec\__init__.py", line 4, in <module>
    from .dictionary import Dictionary
ModuleNotFoundError: No module named 'wikipedia2vec.dictionary'

**Comment by ikuyamada** [comment]
Why does the *examples/text_classification* directory contain a wikipedia2vec directory? As seen in the following URL, the original code does not contain the directory. I think removing the directory resolves the issue.
https://github.com/wikipedia2vec/wikipedia2vec/tree/master/examples/text_classification

**Comment by AMParanoid** [comment]
Issue solved!!!

I runned 'pip uninstall wikipedia2vec && pip install wikipedia2vec -I && rm wikipedia2vec'

The issue was happening before I pasted wikipedia2vec to text_classification. 
I had a similar issue with pytorch. Solved unistallling pytorch and installing with -I.

Now, it's downloading 20newsgroup dataset:

[2019-09-19 20:49:19,121] [INFO] Downloading dataset from https://ndownloader.figshare.com/files/5975967 (14 MB) (_download_20newsgroups@twenty_newsgroups.py:80)

TY!!
